version.js

```diff
@@ -151,16 +151,30 @@ function tryParseVersionBranch(branchName) {
 
 /**
  * Checks the PR to see if the major version should be incremented based on labels.
+ * @param {{ owner: string, repo: string }} repo
  * @param {string} commitMessage
- * @returns {Promise<'minor' | 'major'>}
+ * @returns {Promise<SemVer>}
  */
-async function getIncrementType(commitMessage) {
+async function getDefaultBranchNextVersion(repo, commitMessage) {
+  const previousRelease = await getLatestReleaseVersion(repo);
   const prNumber = tryParsePrNumber(commitMessage);
```

Comment:
Does this function return a _version_ number or a _pull request number_?

I'm not sure how reliable our commit messages are for major version upgrades. When we run them manually, we hand-write the PR title (which then turns into the squash commit's message).

Reply:
I've fixed the docs here now. The `tryParsePrNumber` returns the PR reference from a merge/squash commit message. The PR ref suffix is automatically added by GitHub e.g. `Upgrade terraform-provider-azurerm to v3.107.0 (#2112)` returns `2112`. This is then used to look up the original PR details including the branch name and labels.

Reply:
awesome, thank you for clarifying!